src/lib/tabs/tab-header.ts

```diff
@@ -79,6 +80,7 @@ export const _MdTabHeaderMixinBase = mixinDisableRipple(MdTabHeaderBase);
     'class': 'mat-tab-header',
     '[class.mat-tab-header-pagination-controls-enabled]': '_showPaginationControls',
     '[class.mat-tab-header-rtl]': "_getLayoutDirection() == 'rtl'",
+    '(window:resize)': "_checkPaginationEnabled()"
```

Comment:
We should really just have one library-wide resize handler instead of each component adding their own. Add a TODO?